frontend/resourcebundles/src/main/java/com/virtuslab/gitmachete/frontend/resourcebundles/GitMacheteBundle.java

```diff
@@ -28,7 +28,7 @@ private GitMacheteBundle() {}
    *             but each parameter must be a non-null {@link String} and not just a nullable {@link Object}
    * @return the formatted string
    */
-  public static @PolyTainted String format(@PolyTainted @I18nFormatFor("#2") String format, String @MinLen(1)... args) {
+  public static @PolyTainted String fmt(@PolyTainted @I18nFormatFor("#2") String format, String @MinLen(1)... args) {
```

Comment:
Pls mention why `fmt` and not `format`

Reply:
In javadoc

Reply:
So that someone doesn't attempt to _improve the naming_ in 10 years ;D

Reply:
(if Graphite and/or Sapling don't make this entire project obsolete by then :trollface:)